machine/machine_core/machine.py

```diff
@@ -214,6 +218,29 @@ def get_admin_group(self):
         else:
             return "wheel"
 
+    def get_packages(self, source=False):
```

Comment:
IMHO we should start with putting that logic into imagediff, and only factor it out once we actually have an use case for it.

Reply:
Ya.  I agreed.  I'll move it back.